EIPS/eip-token-standard.md

```diff
@@ -0,0 +1,167 @@
+
+``` js
+function decimals() constant returns (uint8 decimals)
+```
+
+
+#### totalSupply
+
+Returns the total token supply.
+
+``` js
+function totalSupply() constant returns (uint256 totalSupply)
+```
+
+
+
+#### balanceOf
+
+Returns the account balance of another account with address `_owner`.
+
+``` js
+function balanceOf(address _owner) constant returns (uint256 balance)
+```
+
+
+
+#### transfer
+
+Transfers `_value` amount of tokens to address `_to`.
+The command should `throw` if the `_from` account balance has not enough tokens to spend.
+
+
+``` js
+function transfer(address _to, uint256 _value) returns (bool success)
+```
+
+
+
+#### transferFrom
+
+Transfers `_value` amount of tokens from address `_from` to address `_to`.
+
+The `transferFrom` method is used for a withdraw workflow, allowing contracts to transfer tokens on your behalf.
+This can be used for example to allow a contract to transfer tokens on your behalf and/or to charge fees in sub-currencies.
+The command should `throw` unless the `_from` account has deliberately authorized the sender of the message via some mechanism.
+
+``` js
+function transferFrom(address _from, address _to, uint256 _value) returns (bool success)
+```
+
+
+
+#### approve
+
+Allows `_spender` to withdraw from your account multiple times, up to the `_value` amount. If this function is called again it overwrites the current allowance with `_value`.
+
+**NOTE**: To prevent attack vectors like the one [described here](https://docs.google.com/document/d/1YLPtQxZu1UAvO9cZ1O2RPXBbT0mooh4DYKjA_jp-RLM/) and discussed [here](https://github.com/ethereum/EIPs/issues/20#issuecomment-263524729),
+make sure to force users set the allowance to `0` before setting it to another value for the same spender.
+
+``` js
+function approve(address _spender, uint256 _value) returns (bool success)
+```
+
+
+#### allowance
+
+Returns the amount which `_spender` is still allowed to withdraw from `_owner`.
+
+``` js
+function allowance(address _owner, address _spender) constant returns (uint256 remaining)
+```
+
+
+
+### Events
+
+
+#### Transfer
+
+Triggered when tokens are transferred.
+
+``` js
+event Transfer(address indexed _from, address indexed _to, uint256 _value)
+```
+
+
+
+#### Approval
+
+Triggered when `approve(address _spender, uint256 _value)` is called.
+
+``` js
+event Approval(address indexed _owner, address indexed _spender, uint256 _value)
+```
+
+
+
+## Implementation
+
+There are already plenty of ERC20-compliant tokens deployed on the Ethereum network.
+Different implementations have been written by various teams that have different trade-offs: from gas saving to improved security.
+
+#### Example implementations are available at
+- https://github.com/OpenZeppelin/zeppelin-solidity/blob/master/contracts/token/StandardToken.sol
+- https://github.com/ConsenSys/Tokens/blob/master/Token_Contracts/contracts/StandardToken.sol
+
```

Comment:
I would also like to see the MiniMeToken added to this list.  Maintaining historical balance data is key for a number of token use cases.